README.md

```diff
@@ -31,4 +34,62 @@ console.log(v1.minus(v2));
 var v3 = new Vector(3, 4)
 console.log(v3.getLength());
 // => 5
-```
+``` -->
+
+
+1.
+class Human {
+  constructor(name, age) {
+    this.name = name;
+    this.age = age;
+  }
+
+  ageOneYear() {
+    this.age++
+    return `You are now ${this.age} years old`;
+  }
+
+  eating() {
+    return `mmm, mmm, mmm, I'm love'n it`
+  }
+}
+
+let newHu = new Human ('Michell', '25')
+console.log(newHu.age);
+console.log(newHu.ageOneYear())
+console.log(newHu.eating())
+
+
+2.
+class Vector {
+  constructor(x, y) {
+    this.x = x;
+    this.y = y;
+  }
+
+  plus(vec) {
+    let addition = this.x + vec.x;
+    let addition2 = this.y + vec.y;
+    return `x: ${addition} , y: ${addition2}`;
+  }
+
+  minus(vec) {
+    let subtraction = this.y - vec.y;
+    let subtraction2 = this.y - vec.y
+    return `x: ${subtraction} , y: ${subtraction2}`;
+  }
+
+  getLength (nums) {
+    let length = Math.hypot(this.x,this.y)
+    return length;
+  }
+}
+
+
```

Comment:
The plus and minus functions are almost correct. They should return a new Vector instead of a string with the values. Variable names could be slightly clearer.